app/ygnmi/cmd/generator/generator.go

```diff
@@ -45,41 +43,40 @@ func New() *cobra.Command {
 		Args:  cobra.MinimumNArgs(1),
 	}
 
-	generator.Flags().String("schema_struct_path", "", "The Go import path for the schema structs package. If struct generation is enabled, this defaults to base_import_path.")
+	generator.Flags().String("schema_struct_path", "", "The Go import path for the schema structs package. If struct generation is enabled, this defaults to base_package_path.")
```

Comment:
remove "If struct generation is enabled".
Also I'm wondering if we're always generating GoStructs then why is this flag still here? Sorry I might have forgotten some context here.

Reply:
removed, but yes this is flag is needed for our internal use case